helix-core/src/main/java/org/apache/helix/controller/GenericHelixController.java

```diff
@@ -220,17 +221,24 @@ public GenericHelixController(String clusterName, Set<Pipeline.Type> enabledPipe
   class RebalanceTask extends TimerTask {
     final HelixManager _manager;
     final ClusterEventType _clusterEventType;
+    private final boolean _shouldRefreshCache;
     private long _nextRebalanceTime;
 
     public RebalanceTask(HelixManager manager, ClusterEventType clusterEventType) {
       this(manager, clusterEventType, -1);
```

Comment:
Nit: could we create a constant for `REBALANCE_IMMEDIATELY` or `IMMEDIATE_REBALANCE` with value -1?

Reply:
Good idea. But should not be a change in this PR. Let me create an issue for that.

Reply:
https://github.com/apache/helix/issues/640